lib/launcher.rb

```diff
@@ -132,9 +132,7 @@ def launch
       # do nothing. This is to prevent the exception handler below from catching SystemExit exceptions.
       raise
     rescue Exception => e
-      logger.error("Error during startup, terminating. #{e}")
-      logger.debug("#{e}:#{e.backtrace.join("\n")}")
-
+      logger.error("Error during startup, terminating. #{e}", e.backtrace)
```

Comment:
Apologies, I see now that you debug-level messages are being automatically created. Pls. disregard this and other comments about that.